packages/x-charts/src/context/ZoomProvider/useSetupZoom.ts

```diff
@@ -29,27 +29,26 @@ const zoomAtPoint = (
   currentRange: readonly [number, number],
 ) => {
   const [minRange, maxRange] = currentRange;
-  const scaleRatio = scale;
 
   const point = minRange + centerRatio * (maxRange - minRange);
 
-  let newMinRange = (minRange + point * (scaleRatio - 1)) / scaleRatio;
-  let newMaxRange = (maxRange + point * (scaleRatio - 1)) / scaleRatio;
+  let newMinRange = (minRange + point * (scale - 1)) / scale;
+  let newMaxRange = (maxRange + point * (scale - 1)) / scale;
```

Comment:
What you call scale is not exactly a scale. It's the ratio of the new scale divided by the current one.

The previous scale is `100 / (maxRange-minRange)`. If you want the new scale you should do `scaleRatio * (maxRange-minRange) / 100`